admin-dev/themes/new-theme/scss/pages/_orders.scss

```diff
@@ -3,4 +3,18 @@
   .column-shop_name {
     word-break: normal;
   }
+  .order-preview-content {
+    .details th, .product th, .details td, .product td {
+      border: none;
+      padding-top: 3.2px;
+      padding-bottom: 3.2px;
+    }
+    .details td, .product td {
+      padding-top: 0;
+      padding-bottom: 0;
+    }
+    .product th {
+      background-color: #eaebec;
```

Comment:
maybe introduce new color #eaebec in `/var/www/html/PrestaShop/admin-dev/themes/default/sass/modules/_variables.sass` ?